pages/index.js

```diff
@@ -22,6 +22,14 @@ const Index = () => {
         />
       </Head>
       <HomePage />
+      <footer>
+        <p>
+          The contents of this website are deployed from this 
+          <a href="{https://github.com/Real-Dev-Squad/website-members}" target="_blank">
```

Comment:
Do you need `{ }` for html attribute?

Reply:
To render the the href link in react we use { } . Will the link work without {} as well ? Let me try and check in my local

Reply:
Without the { }, no dynamic content will render. But, our string is static anyway :)

Reply:
Yep ok